src/helpers/__tests__/formatBytesString.test.ts

```diff
@@ -0,0 +1,19 @@
+import { formatBytesString } from "../formatBytesString";
+
+describe("formatBytesString", () => {
+    it("format as bytes", () => {
+        expect(formatBytesString(999)).toBe("999 B");
```

Comment:
could we use the same input for all of these? so have the same amount of bytes going in and then it's super clear that the ratios are correct on the other side